presto-product-tests/bin/run_on_docker.sh

```diff
@@ -194,7 +194,11 @@ fi
 trap terminate INT TERM EXIT
 
 # start external services
-EXTERNAL_SERVICES="hadoop-master mysql postgres cassandra sqlserver"
+if [[ "$ENVIRONMENT" == "singlenode-sqlserver" ]]; then
+  EXTERNAL_SERVICES="hadoop-master cassandra sqlserver"
```

Comment:
the comment should link to a tempto issue that - once solved - will allow us to not have cassandra here. If there's not such an issue, create it.